### PR TITLE
Upgrade dependency

### PR DESCRIPTION
**Pull request by donesvad**

### CODE REVIEW

File: .idea/codeStyles/muranoCodeStyle.xml [hunk 1]
```diff
@@ -335,4 +335,11 @@
...
     </arrangement>
   </codeStyleSettings>
+  <codeStyleSettings language="XML">
+    <indentOptions>
+      <option name="INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="2" />
+      <option name="TAB_SIZE" value="2" />
+    </indentOptions>
+  </codeStyleSettings>
 </code_scheme>
```
Comment (alexandrter) [inline .idea/codeStyles/muranoCodeStyle.xml]: New line

File: pom.xml [hunk 2]
```diff
@@ -11,25 +11,24 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.3.4.RELEASE</version>
+    <version>2.7.10</version>
   </parent>
 
   <name>MuranoBot</name>
```
Comment (alexandrter) [inline pom.xml]: let's either remove it or change it to something meaningful.
For me name tag seems pretty [unnecessary ](https://stackoverflow.com/questions/60119816/maven-name-tag-under-project#:~:text=name%3A%20This%20element%20indicates%20the,in%20the%20project%20I%20work.)

File: pom.xml [hunk 3]
```diff
@@ -134,7 +133,7 @@
       </plugin>
     </plugins>
     <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
-<!--      TODO refactor as part of https://trello.com/c/znZmQegs/18-refactor-maven-plugins-in-pomxml-->
+      <!--      TODO refactor as part of https://trello.com/c/znZmQegs/18-refactor-maven-plugins-in-pomxml-->
```
Comment (alexandrter) [inline pom.xml]: task was rejected, remove comment

**Review by alexandrter — COMMENTED** [review]
I see that some maven plugins also have versions. Do we need them? I think they have been automatically generated by maven on project creation. Maybe that's what was meant in [the rejected task](https://trello.com/c/znZmQegs/18-refactor-maven-plugins-in-pomxml). 
So please either upgrade or remove

File: .idea/codeStyles/muranoCodeStyle.xml (hunk 1 above)
Comment (donesvad) [reply .idea/codeStyles/muranoCodeStyle.xml]: If you really want a new line at the end of each file, could you please configure our code style correspondingly.
Don't see any value to add them manually :D

File: pom.xml (hunk 2 above)
Comment (donesvad) [reply pom.xml]: yeah, agree

**Comment by donesvad** [comment]
> I see that some maven plugins also have versions. Do we need them? I think they have been automatically generated by maven on project creation. Maybe that's what was meant in [the rejected task](https://trello.com/c/znZmQegs/18-refactor-maven-plugins-in-pomxml). So please either upgrade or remove

removed, don't see any value in them

File: pom.xml (hunk 3 above)
Comment (donesvad) [reply pom.xml]: done

**Review by alexandrter — APPROVED** [review]

**Pull request merged**